jest.config.js

```diff
@@ -9,10 +9,10 @@ module.exports = {
   collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}', '!<rootDir>/node_modules/'],
   coverageThreshold: {
     global: {
-      branches: 40,
-      functions: 40,
-      lines: 65,
-      statements: 65,
+      branches: 84,
```

Comment:
They're manually generated.  I run a `npm run finish` and the values are outputted.  Since it's a small library, I'm fine with specific values.  It's a forcing function to remember to implement better unit tests.